test/actor.test.js

```diff
@@ -0,0 +1,175 @@
+require('dotenv').config();
+const request = require('supertest');
+const mongoose = require('mongoose');
+const connect = require('../lib/utils/connect');
+const app = require('../lib/app');
+const Actor = require('../lib/models/ActorSchema');
+const Studio = require('../lib/models/StudioSchema');
+const Film = require('../lib/models/FilmSchema');
+
+describe('actor route tests', () => {
+
+  beforeAll(() => {
+    connect();
+  });
+  beforeEach(() => {
+    return mongoose.connection.dropDatabase();
+  });
+
+  afterAll(() => {
+    return mongoose.connection.close();
+  });
+
+  it('can POST a new actor', () => {
+    return request(app)
+      .post('/api/v1/actors')
+      .send({
+        name: 'Havard Forestborn',
+        dob: '2010-06-06',
+        pob: 'Humboldt, CA'
+      })
+      .then(res => {
+        expect(res.body).toEqual({
+          __v: 0,
+          _id: expect.any(String),
+          name: 'Havard Forestborn',
+          dob: '2010-06-06T00:00:00.000Z',
+          pob: 'Humboldt, CA'
+        });
+      });
+  });
+
+  it('can GET all actors', async() => {
+    const actors = await Actor.create([
```

Comment:
What if you added all of your creates to a beforeEach and made them globally accessible? Then you wouldn't have to create a new actor, studio, etc. in an it block every time.